src/helpers/unmountApplication.ts

```diff
@@ -0,0 +1,27 @@
+import { reconciler } from '../core/reconciler';
+import { roots } from '../core/roots';
+import { type Root } from '../typedefs/Root';
+
+export function unmountApplication(root: Root)
+{
+    if (root.internalState.queuedForUnmount)
+    {
+        const fiber = root?.fiber;
+
+        if (fiber)
+        {
+            reconciler.updateContainer(null, fiber, null, () =>
+            {
+                try
+                {
+                    root.applicationState.app.destroy();
+                    roots.delete(root.internalState.canvas!);
+                }
+                catch (error)
+                {
+                    /* ... */
```

Comment:
We should probably at least console.error the error, unless there is an expected error here we're trying to ignore, in which case it should probably be explicitly handled?

Reply:
This try/catch wraps the `app.destroy()`, which may be called on an app that has already been destroyed. Pixi.js doesn't emit any useful errors in this case, so we're just eating the error.

The best solution here would probably be an upstream change to Pixi.js to throw a useful error when attempting to destroy an app that's already been destroyed, then we could handle that error appropriately. Otherwise, there's not (as far as I'm aware) a reliable way of determining whether an app has been destroyed.

Reply:
Perhaps check if the application has a stage? That's created at construction time and is nulled out when destroyed.